nikola/conf.py.in

```diff
@@ -645,6 +645,21 @@ COMMENT_SYSTEM_ID = ${COMMENT_SYSTEM_ID}
 # Enable comments on picture gallery pages?
 # COMMENTS_IN_GALLERIES = False
 
+# Use a thumbnail (defined by ".. previewimage:") in list of galleries
+# for each gallery
+GALLERIES_USE_THUMBNAIL = True
+
+# Columns of thumbnails list in galleries index
+GALLERIES_COLUMNS = 3
```

Comment:
How do you calculate "How many columns?" based on the THUMBNAIL_SIZE? I used that setting to put a `<div class="row">` or a `<div class="col-md-4">`.

I know *nothing* about Bootstrap, so if you know another better way to do that, please let me know.